### ISSUE TITLE
Instance of player in world class causing exception

### ISSUE THREAD

**Issue by estarkey7** [estarkey7/CSIS3700_Project2#4]
When I try to run the latest code, I get an exception at line 51 in the world class when it tries to make a new player. I am almost done with the enemy and coin classes but I can't test them.

**Comment by nmroncone** [comment]
I think it might of been because I added some images to the image folder that still needed synced. I just synced them now. hopefully that was it.

Thanks,
            Nick Roncone

On Apr 29, 2015, at 11:22 PM, Ed Starkey <notifications@github.com<mailto:notifications@github.com>> wrote:

When I try to run the latest code, I get an exception at line 51 in the world class when it tries to make a new player. I am almost done with the enemy and coin classes but I can't test them.

##

**Comment by estarkey7** [comment]
Yeah, that fixed it. Game looks awesome!! I'm hoping to get the bugs finished tonight. Do we have a tree quiz tomorrow?

**Comment by nmroncone** [comment]
I hope not. I thought we didn't have class tomorrow because I remember him saying that last Thursday was our last lab. I honestly have no idea though now.

Thanks,
            Nick Roncone

On Apr 30, 2015, at 12:24 AM, Ed Starkey <notifications@github.com<mailto:notifications@github.com>> wrote:

Yeah, that fixed it. Game looks awesome!! I'm hoping to get the bugs finished tonight. Do we have a tree quiz tomorrow?

##

**Comment by nmroncone** [comment]
Hey Ed, I think you need to include the enemy files in the commit because it's saying its missing enemy.cpp

Thanks,
            Nick Roncone

On Apr 30, 2015, at 12:24 AM, Ed Starkey <notifications@github.com<mailto:notifications@github.com>> wrote:

Yeah, that fixed it. Game looks awesome!! I'm hoping to get the bugs finished tonight. Do we have a tree quiz tomorrow?

##
